code/modules/mob/dead/observer/observer.dm

```diff
@@ -204,8 +204,8 @@
 	. = ..()
 
 	if(statpanel("Stats"))
-		if(EvacuationAuthority)
-			var/eta_status = EvacuationAuthority.get_status_panel_eta()
+		if(SSevacuation)
+			var/eta_status = SSevacuation.get_status_panel_eta()
```

Comment:
```suggestion
			var/eta_status = SSevacuation?.get_status_panel_eta()
```
You can then remove the `if(SSevacuation)`. since `eta_status` will be checked for null values right next.